deploy/scripts/mainnet/0013-CarbonController-upgrade.ts

```diff
@@ -0,0 +1,22 @@
+import { DeployedContracts, InstanceName, setDeploymentMetadata, upgradeProxy } from '../../../utils/Deploy';
+import { DeployFunction } from 'hardhat-deploy/types';
+import { HardhatRuntimeEnvironment } from 'hardhat/types';
+
+/**
+ * @dev trade by source extended input upgrade
```

Comment:
This will require a storage slot change, so we'd rather keep it.
We could replace it with another contract that has similar storage layout but it's an unnecessary risk.

Reply:
fixed